packages/snap/src/generate-snapshots/filename-utils.ts

```diff
@@ -1,7 +1,5 @@
-import {basename} from 'path';
-
 export interface IFileParts {
-  base: string;
+  file: string;
```

Comment:
You don't seem to be using this field.

Reply:
Probably Yagni... just wanted to leave it in in case